src/theme/colors.ts

```diff
@@ -0,0 +1,26 @@
+export const colors = {
+  black: '#000000',
+  darkGrey: '#5B5B5B',
+  mediumGrey: '#E1E1E1',
+  lightGrey: '#EFEFEF',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'lightGrey' is not sorted alphabetically](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)